ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-create-test.js

```diff
@@ -38,101 +30,1067 @@ module('Acceptance | kv-v2 workflow | secret and version create', function (hook
 
   module('admin persona', function (hooks) {
     hooks.beforeEach(async function () {
-      const token = await runCmd([
-        createPolicyCmd('admin', adminPolicy(this.backend)),
-        createTokenCmd('admin'),
-      ]);
+      const token = await runCmd(tokenWithPolicyCmd('admin', personas.admin(this.backend)));
       await authPage.login(token);
     });
-    test.skip('create new root secret flow', async function (assert) {
-      assert.expect(0);
-      // TODO: navigate to kv backend from backends
-      // ✅ shows list items / no empty state
-      // ✅ shows correct toolbar options
-      // click "Create secret"
-      // ✅ URL is vault/secrets/:backend/kv/create
-      // ✅ Path input is empty
-      // click [data-test-kv-save]
-      // ✅ Shows validation errors
+    test('cancel on create clears model (a)', async function (assert) {
+      const backend = this.backend;
+      await visit(`/vault/secrets/${backend}/kv/list`);
+      assert.dom(PAGE.list.item()).exists({ count: 1 }, 'single secret exists on list');
+      assert.dom(PAGE.list.item('app/')).hasText('app/', 'expected list item');
+      await click(PAGE.list.createSecret);
+      await fillIn(FORM.inputByAttr('path'), 'jk');
+      await click(FORM.cancelBtn);
+      assert.dom(PAGE.list.item()).exists({ count: 1 }, 'same amount of secrets');
+      assert.dom(PAGE.list.item('app/')).hasText('app/', 'expected list item');
+      await click(PAGE.list.createSecret);
+      await fillIn(FORM.inputByAttr('path'), 'psych');
+      await click(PAGE.breadcrumbAtIdx(1));
+      assert.dom(PAGE.list.item()).exists({ count: 1 }, 'same amount of secrets');
+      assert.dom(PAGE.list.item('app/')).hasText('app/', 'expected list item');
+    });
+    test('cancel on new version rolls back model (a)', async function (assert) {
```

Comment:
👏  yay! nice to see a bug is fixed in the new work and no more floating ghost models 👻

Reply:
jk this was when creating a new secret and cancel brought back to the LIST view